src/service/wrapped-service.ts

```diff
@@ -0,0 +1,333 @@
+/**
+ *  SudoSOS back-end API service.
+ *  Copyright (C) 2024  Study association GEWIS
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published
+ *  by the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ *  @license
+ */
+import WrappedResponse from '../controller/response/wrapped-response';
+import WithManager from '../database/with-manager';
+import Wrapped from '../entity/wrapped';
+import User from '../entity/user/user';
+import Transaction from '../entity/transactions/transaction';
+import SubTransaction from '../entity/transactions/sub-transaction';
+import SubTransactionRow from '../entity/transactions/sub-transaction-row';
+import ProductRevision from '../entity/product/product-revision';
+import { Between, In } from 'typeorm';
+
+export interface UpdateWrappedParameters {
+  ids?: number[],
+}
+
+export default class WrappedService extends WithManager {
+  public async getWrappedForUser(userId: number): Promise<WrappedResponse | null> {
```

Comment:
Also, this is not really your fault, but I try to work more towards this goal:
https://github.com/GEWIS/sudosos-backend/issues/153

and that returning a `Wrapped` and having the controller turn the `Wrapped` into a `WrappedResponse` is prefered since this makes the WrappedService more flexible.